src/renderer/Layout.tsx

```diff
@@ -128,6 +129,21 @@ const Layout: React.FC<IProps> = (props: IProps) => {
       fill: true,
       inactivityTimeout: 0,
       playbackRates: [0.25, 0.5, 1, 1.5, 2],
+      plugins: {
+        hotkeys: {
+          seekStep: 10,
+          enableModifiersForNumbers: false,
+          forwardKey(event: { which: number }) {
+            return event.which === 76 || event.which === 39;
```

Comment:
I think the 74, 75 and 75 are just ASCII here. What's the 32, 37, 39 etc? 

Didn't see a reference for what this actually is on the videojs-hotkeys page, probably could do with a comment.